source/agora/consensus/data/Transaction.d

```diff
@@ -450,6 +446,50 @@ unittest
     }
 }
 
+/// Transaction type serialize & deserialize for unittest
+unittest
+{
+    Transaction payment_tx = Transaction(
+        TxType.Payment,
+        [Input(Hash.init, 0)],
+        [Output.init]
+    );
+    assert(agora.common.Deserializer.deserialize!Transaction(serializeFull(payment_tx)) == payment_tx);
```

Comment:
Why do you use the FQN ?

Reply:
FQN = Fully Qualified Name, e.g. `agora.common.Deserializer.deserialize` instead of just `deserialize`.

Reply:
The same scope has a function with the same name.

Reply:
That's why it was a good idea to move the unittests outside the class itself. Which is what you did here, which was good. But we just need to rename it again.